conf/init.go

```diff
@@ -110,9 +102,9 @@ type syslog struct {
 	Host string
 }
 
-func GetEtcdAPI() client.KeysAPI{
-	return client.NewKeysAPI(etcdClient)
-}
+//func GetEtcdAPI() client.KeysAPI{
```

Comment:
If we do not need this code, we can delete them directly.

Reply:
ok，I will Unified coding style  #42